application/src/pages/charte-graphique.astro

```diff
@@ -0,0 +1,262 @@
+              ))
+            }
+          </ul>
+        </div>
+      </div>
+    </div>
+
+    <!-- Typography -->
+    <div class="mb-16">
+      <h2 class="text-3xl md:text-4xl font-bold mb-8 text-gray-900 dark:text-white">Typographie</h2>
+      <div class="grid md:grid-cols-2 gap-8">
+        {
+          typography.map((font) => (
+            <div class="bg-white dark:bg-gray-800 rounded-lg p-8 border border-gray-200 dark:border-gray-700">
+              <h3 class="text-2xl font-bold mb-4 text-gray-900 dark:text-white" style={`font-family: "${font.family}"`}>
+                {font.family}
+              </h3>
+              <p class="text-gray-600 dark:text-gray-400 mb-4">{font.usage}</p>
+              <div class="space-y-2 text-sm text-gray-600 dark:text-gray-400">
+                <p>
+                  <strong>Graisses disponibles :</strong> {font.weights.join(', ')}
+                </p>
+                {font.fallback && (
+                  <p>
+                    <strong>Fallback :</strong> {font.fallback}
+                  </p>
+                )}
+              </div>
+              <div class="mt-6 space-y-2">
+                {font.weights.slice(0, 4).map((weight) => (
+                  <p
+                    class="text-gray-900 dark:text-white"
+                    style={`font-family: "${font.family}"; font-weight: ${weight}`}
+                  >
+                    The quick brown fox jumps ({weight})
+                  </p>
+                ))}
+              </div>
+            </div>
+          ))
+        }
+      </div>
+    </div>
+
+    <!-- Download Center -->
+    <div class="mb-16">
+      <h2 class="text-3xl md:text-4xl font-bold mb-8 text-gray-900 dark:text-white">Centre de téléchargement</h2>
+      <div class="bg-blue-50 dark:bg-blue-900/20 rounded-lg p-8 border border-blue-200 dark:border-blue-800">
+        <h3 class="text-xl font-bold mb-4 text-blue-900 dark:text-blue-100">Assets officiels</h3>
+        <p class="text-gray-700 dark:text-gray-300 mb-6">
+          Téléchargez les logos et assets officiels de Hoverkraft dans différents formats. Pour toute question ou
+          demande spécifique, contactez-nous.
+        </p>
+        <div class="grid md:grid-cols-2 gap-4">
+          <a
+            href="#"
+            class="inline-flex items-center justify-center px-6 py-3 bg-blue-600 hover:bg-blue-700 text-white rounded-lg font-medium transition-colors"
+            onclick="alert('Le pack de logos sera bientôt disponible au téléchargement'); return false;"
+          >
+            <svg class="w-5 h-5 mr-2" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+              <path
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                stroke-width="2"
+                d="M4 16v1a3 3 0 003 3h10a3 3 0 003-3v-1m-4-4l-4 4m0 0l-4-4m4 4V4"></path>
+            </svg>
+            Pack de logos (ZIP)
+          </a>
+          <a
+            href="#"
```

Comment:
Complete guide should be also retrieve from https://github.com/hoverkraft-tech/branding/blob/main/branding-guideline-pdf, in order to be downloadable from landign-page